.github/workflows/prettier-check.yml

```diff
@@ -0,0 +1,31 @@
+name: autofix.ci
+on:
+  push:
+    branches:
+      - trunk
```

Comment:
If it's clean on a PR, then I don't think this should ever have to run on `trunk`